### ISSUE TITLE
regular expression /a[\s\S]b/

### ISSUE THREAD

**Issue by Kvark161** [plusvic/yara#249]
rule_file:
rule R {
strings:
    $a = /a[\s\S]b/
condtion:
    $a
}
file: 
a  b
$a is not found

**Comment by plusvic** [comment]
The same is happening with:

```
rule R {
  strings:
    $a = /a[\d\D]b/
  condition:
    $a
}
```
file: 

```
a1b
```

**Comment by plusvic** [comment]
Fixed in b6bccf3c620d126d5f43b7ef4ea93dfe22c2fd00
